apps/server/src/shared/repo/task/task.repo.ts

```diff
@@ -14,7 +14,14 @@ export class TaskRepo extends BaseRepo<Task> {
 
 	async findById(id: EntityId): Promise<Task> {
 		const task = await super.findById(id);
-		await this._em.populate(task, ['course', 'lesson', 'submissions']);
+		await this._em.populate(task, [
```

Comment:
I want to outsource this both populates, but find no way that mikroORM is fine with the type.